src/views/RegisterView.vue

```diff
@@ -123,16 +124,19 @@ export default {
         }
         return `Minimum username length is ${usernameLength} characters`;
       }
+      return null;
```

Comment:
did it also add these?

Reply:
Again this was me. The new eslint version picks up more stuff.